eng/pipelines/sync-sharedcode.yml

```diff
@@ -9,42 +9,36 @@ schedules:
     - feature/v3
   always: true
 
-resources:
-  repositories:
-    - repository: azure-sdk-tools
-      type: github
-      name: Azure/azure-sdk-tools
-      endpoint: azure
-      ref: refs/tags/azure-sdk-tools_20240206.1
-
-variables:
-- group: Release Secrets for GitHub
```

Comment:
Is this variable group unused now?

Reply:
It's used here: https://github.com/Azure/autorest.csharp/pull/4423/files#diff-bea2798fbc59869a31fcc226196bc6eb49784b4ab5a8d3793dd17dcd3baa9a04R280

The variable group is now directly associated with the pipeline: https://dev.azure.com/azure-sdk/internal/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=1375&view=Tab_Variables